green_cli/common.py

```diff
@@ -355,6 +358,7 @@ def getbalance(session, details):
 @click.option('--num-confs', default=0, expose_value=False, callback=details_json)
 @click.option('--all-coins', type=bool, default=False, expose_value=False, callback=details_json)
 @click.option('--expired-at', type=int, expose_value=False, callback=details_json)
+@click.option('--dust-limit', type=int, expose_value=False, callback=details_json)
```

Comment:
Should we add `--dust-limit` to `createtransaction` as well?

Reply:
I will look at that once I've implemented it for create_transaction, its not there yet.